src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TeleporterListener.java

```diff
@@ -64,7 +64,7 @@ public void onPressurePlateEnter(PlayerInteractEvent e) {
             // Pressure plate was an elevator
             ElevatorPlate elevator = SlimefunItems.ELEVATOR_PLATE.getItem(ElevatorPlate.class);
             elevator.openInterface(p, b);
-        } else if (item instanceof AbstractTeleporterPlate && ((AbstractTeleporterPlate) item).hasAccess(p, b)) {
+        } else if (item instanceof AbstractTeleporterPlate abstractTeleporterPlate && abstractTeleporterPlate.hasAccess(p, b)) {
```

Comment:
We do not always have to be so literal about it.
```suggestion
        } else if (item instanceof AbstractTeleporterPlate teleporterPlate && teleporterPlate.hasAccess(p, b)) {
```